crates/storage_models/src/merchant_connector_account.rs

```diff
@@ -28,6 +28,7 @@ pub struct MerchantConnectorAccount {
     pub payment_methods_enabled: Option<Vec<serde_json::Value>>,
     pub connector_type: storage_enums::ConnectorType,
     pub metadata: Option<pii::SecretSerdeValue>,
+    pub frm_configs: Option<Secret<serde_json::Value>>, //Option<FrmConfigs>
```

Comment:
Hey @rishavkar, I understand this is a PR that's already merged, but wanted to bring this to your attention.

Whenever you're adding fields to a struct corresponding to a database table, please add it at the last, so that it is in correspondence with how the database has it. When we do a `diesel migration run`, Postgres adds the column to the end of the table, and `diesel` updates the `schema.rs` file to match the database table state, thus adding `frm_configs` to the end. Because of how `diesel` handles queries, it expects the column order in `schema.rs` and the field order in the struct here to be the same. If that is not the case, we'd end up having unexpected behavior (such as data for one column being populated in another) when running our application.